File: practise/python/p3.py
# 3. Find the Largest of Three Numbers
a = int(input("Enter first number: "))  # Read first integer from user
b = int(input("Enter second number: "))  # Read second integer from user
c = int(input("Enter third number: "))  # Read third integer from user

if a > b and a > c:  # Check if 'a' is greater than both 'b' and 'c'
    print(a, "is the largest")  # Print result
elif b > c:  # Check if 'b' is greater than 'c'
    print(b, "is the largest")  # Print result
else:  # If neither 'a' nor 'b' is the largest, then 'c' must be the largest
    print(c, "is the largest")  # Print result

File: practise/c/p3.c
// 3. Find the Largest of Three Numbers
#include <stdio.h>

int main() {
    int a, b, c;  // Declare three integer variables
    printf("Enter three numbers: ");  // Prompt user for input
    scanf("%d %d %d", &a, &b, &c);   // Read three integers from user

    if(a > b && a > c)   // Check if 'a' is greater than both 'b' and 'c' logical
        printf("%d is the largest\n", a);   // Print result
    else if(b > c)   // Check if 'b' is greater than 'c'
        printf("%d is the largest\n", b);   // Print result
    else  // If neither 'a' nor 'b' is the largest, then 'c' must be the largest
        printf("%d is the largest\n", c);   // Print result

    return 0;
}
